doc/nixcon.md

```diff
@@ -43,3 +43,74 @@ Organisers are using:
+- Linked logo on the website
+- Linked shout-out or repost from official social media accounts
+- Possibility to distribute your own stickers/swag/merch at the venue
+- Tickets: 1+ corporate tickets[^1]
+
+[^1]: One corporate ticket per 1024 EUR. Only corporate tickets come with company name/logo recognition on the badge. Corporate tickets can also be bought individually when ticket sales open.
+
+### Silver (2048+ EUR)
+- Everything from Bronze
+- Shout-out in the opening
+- Slot on break slideshow (provide a slide in advance)
+- Possibility to distribute flyers (provide them in advance) to all attendees in bag
+- Tickets: 2+ corporate tickets[^1]
+
+### Gold (4096+ EUR):
+- Everything from Silver
+- 5 minute lightning talk slot
+- Dedicated space for a booth[^2]
+- Tickets: 4+ corporate tickets[^1]
+
+[^2]: These perks may be limited in availability.
+
+### Diamond (8192+ EUR):
+- Everything from Gold
+- Logo on some official conference swag[^2][^3]
+- 1 minute presentation slot (provide the slides in advance) in the conference opening
+- Intro of all recordings show your name/logo
+- Possibility to distribute items (provide them in advance) to all attendees in bag
+- Tickets: 8+ corporate tickets[^1]
+
+[^3]: These perks require production and cannot be guaranteed if too late.
+
+### General info
+- Sponsors can be tier-limited or rejected by the Nix Steering Committee for any reason.
+- The outline of all sponsor content must be disclosed in advance. We reserve the right to reject content.
+- Visibility priority is given according to tiers and donation amount.
+- The sponsor can selectively abstain from perks.
+- Surplus funds will be used to support official Nix projects.
+- Reach out if you'd like to sponsor in a more custom way, such as food, drinks, services, equipment, or only specific perks.
+
+### How to sponsor
+
+Send an email to sponsor@nixos.org (which will forward to both the Nix Steering Committee and the NixOS Foundation board) with:
+- Company name and website
+- Desired tier, amount and perks
+- Any other details
+
+## Sponsorship process
+
+The following is internal documentation and should not go on the website. For 2025 it involves the following respondents:
+- Board respondent: @ra33it0 (deputy: @infinisil)
+- SC respondent: @tomberek (deputy: @winterqt)
+
+1. After receiving the email, board respondent replies that the email was received and that they will hear back within about 2 weeks.
+2. SC respondent ensures the SC internally rejects or provisionally approves the sponsor within 1 week.
+   - If rejected, SC respondent replies to the sponsor with the rejection message.
+3. If provisionally approved by the SC, SC respondent forwards the name and website of the sponsor to the NixCon orga Matrix room for organiser approval.
+4. Within 1 week, the SC respondent checks whether organisers are okay with the sponsor, and if not, ensures that the SC internally reevaluates final rejection or approval of the sponsor.
```

Comment:
What does the second part mean?

Reply:
if organizers are not OK, SC has to reevaluate and either accept that veto or ignore it (although ignoring it would be not good :))

Reply:
In other words: The organisers have a chance to give input to the SC before a sponsor is approved. It's still the SC's final call, which can depend on how strong the disagreement is among organisers.

Reply:
And it's the SC respondents (@tomberek) task to determine whether there's any disagreement at all among organisers, because if there's no disagreement at all (which I expect to be the case for most sponsors), there's no need to bring that sponsor up again with the entire SC.

Reply:
Thanks! That clarifies it